src/components/Welcome.vue

```diff
@@ -76,6 +92,20 @@ export default {
       return seenTerritories.filter(s => !(this.territories && this.territories.some(t => t.id === s.id)));
     },
   },
+  methods: {
+    advertiseMsg() {
+      const readByUser = JSON.parse(localStorage.getItem('readByUser'));
+      if (!readByUser) {
```

Comment:
there's something missing in this logic. If the user never clicks the close button, the next time they open the app, the banner should still show.

Reply:
okay, just pushed to fix that. i originally thought we were just doing a 'view once' type of banner.